@navikt/core/tokens/package.json

```diff
@@ -19,8 +19,11 @@
   ],
   "scripts": {
     "build": "tsx ./config/build.ts > /dev/null && tsx ./config/version-tag.ts && yarn build:darkside",
-    "build:darkside": "tsx ./darkside && yarn build:figma-config",
+    "build:darkside": "tsx ./darkside && yarn build:figma-config && yarn build:plugin",
     "build:figma-config": "tsx ./darkside/figma",
+    "build:plugin-dev": "esbuild darkside/figma/plugin/plugin.ts --target=es2016 --bundle --define:process.env.NODE_ENV=\\\"development\\\" --outfile='darkside/figma/plugin/plugin.js'",
+    "build:plugin": "esbuild darkside/figma/plugin/plugin.ts --target=es2016 --bundle --define:process.env.NODE_ENV=\\\"production\\\" --outfile='darkside/figma/plugin/plugin.js'",
+    "build:plugin-watch": "esbuild darkside/figma/plugin/plugin.ts --target=es2016 --bundle --define:process.env.NODE_ENV=\\\"development\\\" --outfile='darkside/figma/plugin/plugin.js' --watch",
```

Comment:
Can these "scripts" be refactored somehow?

For example, if we change `process.env.NODE_ENV === "production"` to `process.env.NODE_ENV !== "development"` we _could_ do this (I think):
```
    "build:plugin": "esbuild darkside/figma/plugin/plugin.ts --target=es2016 --bundle --outfile='darkside/figma/plugin/plugin.js'",
    "build:plugin-dev": "yarn build:plugin --define:process.env.NODE_ENV=\\\"development\\\"",
    "build:plugin-watch": "yarn build:plugin-dev --watch",
```

You could also consider if we need `build:plugin-watch` depending on how often you think we will use it.

Reply:
Removed watch 👍 

This was actually a deliberate choice since it's a little ambiguous if esbuild uses production or development based on some flags. So to be safe it's easier to manually declare them

> When using the [build](https://esbuild.github.io/api/#build) API, all process.env.NODE_ENV expressions are automatically [defined](https://esbuild.github.io/api/#define) to "production" if all [minification](https://esbuild.github.io/api/#minify) options are enabled and "development" otherwise. This only happens if process, process.env, and process.env.NODE_ENV are not already defined. This substitution is necessary to avoid React-based code crashing instantly (since process is a node API, not a web API).

https://esbuild.github.io/api/#platform